go-hello/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func main() {
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "<h1>Hello World from Go,</h1>\n<p>You've requested: %s</p>\n", r.URL.Path)
+	})
+
+	http.ListenAndServe(":8080", nil)
```

Comment:
Should be `http.ListenAndServe(":"+os.Getenv("PORT"), nil)`, since cloud foundry assigns a random port and passes it to the application via environment. Have you verified that this works on cloud.gov?

Reply:
I did test it in Cloud.gov.  Odd that it worked with :8080.  Made the requested change, and tested it again in Cloud.gov.